src/main/java/com/redi/shortener/services/GreetingService.java

```diff
@@ -2,12 +2,13 @@
 
 import com.redi.shortener.model.CreateGreetingRequest;
 import com.redi.shortener.model.Greeting;
+import java.util.Locale;
 import java.util.UUID;
 import org.springframework.stereotype.Service;
 
 @Service
 public class GreetingService {
   public Greeting create(final CreateGreetingRequest request) {
-    return new Greeting(UUID.randomUUID(), request.name());
+    return new Greeting(UUID.randomUUID(), request.name().toUpperCase(Locale.ROOT));
```

Comment:
What was the reason behind this change? It breaks the test: https://github.com/RediJavaClassroom/url-shortener-Shivamanesh/actions/runs/3307300852/jobs/5458830567#step:4:9285

I would suggest to revert it, or change the test so that it will match on the name in the capital case